src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -0,0 +1,40 @@
+module FileConventions.Test
+
+open System
+open System.IO
+open FileConventions
+open NUnit.Framework
+open NUnit.Framework.Constraints
+
+[<SetUp>]
+let Setup () =
+    ()
+
+[<Test>]
+let HasCorrectShebangTest1 () =
+    let fileInfo = (FileInfo $"{__SOURCE_DIRECTORY__}{Path.DirectorySeparatorChar}DummyFiles{Path.DirectorySeparatorChar}DummyWithoutShebang.fsx")

```

Comment:
@realmarv forgot to say, when constructing absolute paths, it's easier and shorter to use Path.Combine(foo,bar,baz) instead of concatenating Path.DirectorySeparatorChar many times.

Reply:
Great thanks!